docs/user-guide/content-sources.rst

```diff
@@ -98,6 +103,7 @@ The pulp-admin client can be use to list all defined content sources as follows:
   Base URL:       http://content-world/content/
   Enabled:        1
   Expires:        3d
+  Headers:        X-RHUI-ID=1234 \ X-Csrf-Token=31415926535897
```

Comment:
The trailing slash is unnecessary per configparser's [long header field](https://tools.ietf.org/html/rfc822.html#section-3.1.1)
I removed the `\` from the example, but did not remove the checks for `\` at https://github.com/pulp/pulp/blob/master/server/pulp/server/content/sources/model.py#L253 and
https://github.com/pulp/pulp/pull/3018/files#diff-83a560fd6552be2c1c0b06503804c190R159
for users who are using the old config format.

As far as showing the sources on separate lines, the newline character is deliberately replaced by a `' '` by pulp-admin at https://github.com/pulp/pulp/blob/master/client_lib/pulp/client/extensions/core.py#L366
I've updated this to a v = v.replace('\n', '\n\t\t') to keep the newline and tab it for formatting